src/status_im/feature_flags.cljs

```diff
@@ -20,7 +20,8 @@
     ::wallet.graph                       (enabled-in-env? :FLAG_GRAPH_ENABLED)
     ::wallet.import-private-key          (enabled-in-env? :FLAG_IMPORT_PRIVATE_KEY_ENABLED)
     ::wallet.long-press-watch-only-asset (enabled-in-env? :FLAG_LONG_PRESS_WATCH_ONLY_ASSET_ENABLED)
-    ::wallet.wallet-connect              (enabled-in-env? :FLAG_WALLET_CONNECT_ENABLED)}))
+    ::wallet.wallet-connect              (enabled-in-env? :FLAG_WALLET_CONNECT_ENABLED)
+    ::wallet.swap                        (enabled-in-env? :FLAG_SWAP_ENABLED)}))
```

Comment:
this map is alpha sorted, let's keep it that way :)

Reply:
Updated!